core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -131,9 +131,9 @@
   public static final String MESSAGE_ACCEPT_ENCODING = "grpc-accept-encoding";
 
   /**
-   * The default maximum uncompressed size (in bytes) for inbound messages. Defaults to 100 MiB.
+   * The default maximum uncompressed size (in bytes) for inbound messages. Defaults to 4 MiB.
```

Comment:
Two questions (as a user) I have when looking at this:  

- Why did the gRPC team pick 4MB?  It was 100MB before, why is it 4MB now?  Why not 4MB + 1 ?
- Is it okay for me to change this in my application?  I *think* I want a larger size, but the gRPC team is being conservative with a 4MB window.  

Having a rationale explains the answer to both of these questions.  Why did we pick it, and what would be a good reason to change it.